test/e2e/data/infrastructure-vsphere/kustomization/conformance/worker-node-size.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: VSphereMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-worker
+  namespace: ${NAMESPACE}
+spec:
+  template:
+    spec:
+      cloneMode: linkedClone
```

Comment:
Is this specific to Vsphere? @srm09 and @aartij17 ?

Reply:
The template is specific to vSphere. Does that answer your question?